service/plugins/perf/static/js/perf.js

```diff
@@ -604,12 +642,24 @@ function TestSample(sample){
   }
 }
 
-var drawTrendGraph = function(trendSamples, tests, scope, taskId, compareSamples) {
+var drawTrendGraph = function(
+  trendSamples, tests, scope, taskId, compareSamples, PerfChartService
```

Comment:
would it make more sense to have the parameters just be scope and PerfChartService?